cmd/ftl/cmd_serve.go

```diff
@@ -15,8 +14,10 @@ import (
 	"time"
 
 	"connectrpc.com/connect"
+	"github.com/XSAM/otelsql"
 	"github.com/alecthomas/types/optional"
 	_ "github.com/jackc/pgx/v5/stdlib" // pgx driver
+	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
```

Comment:
We have a test  in internal/observability/client_test.go to ensure the semconv version is consistent that you'll need to update to mention this file, _or_ alternatively change the wording to be more generalised.

Probably now there are N locations we should just make the wording generalised.

Reply:
done!